packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -130,7 +131,7 @@ export class ListItem
    *
    * @private
    */
-  @property() dragHandle = false;
+  @property({ reflect: true }) dragHandle = false;
```

Comment:
Nice catch on these. Was the lack of reflection causing E2E or screenshot test failures? If not, can you restore these to keep your PR focused. We'll restore these separately.

Reply:
No test failures that I saw.